### DISCUSSION TITLE
error : Multiple production instances

### DISCUSSION THREAD

**Discussion by xeac** [pimcore/pimcore#17594] [Q&A / Support] [answered]
hi all,

Can someone help me to resolve the issue "Multiple production instances". 
![image](https://github.com/user-attachments/assets/c258b027-6e98-465d-8055-c392aaf60801)

I have three instances but non of them is set as Production in config.yaml or .env.

Do you have any idea how can i resolve this message?

thanks

**Comment by ghost** [comment, marked as answer]
@xeac Hello, this can be solved only by the Pimcore Enterprise support. Please raise a ticker here: https://pimcore.atlassian.net/servicedesk/customer/portal/2 In case you don't have access there, please contact your sales or partner manager. Thanks!

**Reply by xeac** [reply]
Many thanks. Will reach out to the support.
